src/CMakeLists.txt

```diff
@@ -439,6 +439,9 @@ if (TARGET ch_contrib::avrocpp)
     dbms_target_link_libraries(PRIVATE ch_contrib::avrocpp)
 endif ()
 
+set_source_files_properties(Common/CARESPTRResolver.cpp PROPERTIES COMPILE_FLAGS -Wno-reserved-identifier)
```

Comment:
Why?

Reply:
Because it includes c-ares headers and they won't build with `reserved-identifier`